app/Main.hs

```diff
@@ -1,6 +1,17 @@
 module Main where
 
-import Lib
+import Control.Monad (join)
+
```

Comment:
I had read somewhere some people will separate it out into 3 sections - stdlib stuff, 3rd party libs, and internal libs. Thoughts on that?